BUILDING.md

```diff
@@ -69,9 +69,8 @@ There are three support tiers:
 
 ### Supported platforms
 
-The community does not build or test against end-of-life distributions (EoL).
-Thus, we do not recommend that you use Node.js on end-of-life or unsupported
-platforms in production.
+The community does not build or test against end-of-life distributions (EoL). Do
+not use Node.js on end-of-life or unsupported platforms in production.
```

Comment:
IMHO we can't have a `Do not` here.
```suggestion
Use of Node.js on EoL platforms is unsupported.
```

Reply:
I think `Do not` is appropriately more forceful and FWIW avoids passive voice.

If we really want to avoid `Do not` here, maybe change it to a `Do` as this?: `For production purposes, use only supported platforms.`?

Reply:
> `For production purposes, use only supported platforms.`?

I like this better, but I prefer a statement that has Node.js as the object. (using an EoL platform in production has $REASONS which might span beyond the scope of Node.js use)
Refs: https://github.com/nodejs/node/pull/12672
Refs: https://github.com/nodejs/build/issues/688

Reply:
How about this?: `For high-value production applications, run Node.js on supported platforms only.`

Reply:
👍 
(Since Node.js is the object, you could also drop the `high-value`):